indico_themes_canonical/static/js/default.js

```diff
@@ -99,19 +102,74 @@ function addMetaTag() {
   const meta = document.createElement("meta");
   meta.name = "viewport";
   meta.content = "width=device-width,initial-scale=1.0";
-  document.getElementsByTagName('head')[0].appendChild(meta);
+  document.getElementsByTagName("head")[0].appendChild(meta);
 }
 
 /* Moves the header nav to be within the stylised header */
-function moveHeaderNav () {
+function moveHeaderNav() {
   const headerCont = document.querySelector(".confheader");
   const headerNav = document.querySelector(".header");
   headerCont.prepend(headerNav);
 }
 
+/* Adds counts to the head cells of the registrations table */
+
+function addCountsToTableHead() {
+  // Only run this function if we are on the registrations page which contains
+  // the ending path of /participants
+  if (window.location.pathname.endsWith("/participants")) {
+    const table = document.querySelector(".i-table");
+    if (!table) {
+      return;
+    }
+    const counts = countContent("Yes", table);
+    renderCounts(counts, table);
+  }
+}
+
+function countContent(content, table) {
+  const rows = table.querySelectorAll("tbody tr");
+  const counts = {};
+
+  rows.forEach((row) => {
+    const cells = row.querySelectorAll("td");
+    cells.forEach((cell, index) => {
+      const cellText = cell.innerText;
+      if (cellText === content) {
+        const headCell = table.querySelector(
+          `thead th:nth-child(${index + 1})`
+        );
+        const headCellText = headCell.innerText;
+        if (headCellText in counts) {
+          counts[headCellText]++;
+        } else {
+          counts[headCellText] = 1;
+        }
+      }
+    });
+  });
+
+  return counts;
+}
+
+function renderCounts(counts, table) {
+  const headCells = table.querySelectorAll("thead th");
+  const headCellCount = headCells.length;
+
+  for (let i = 0; i < headCellCount; i++) {
+    const headCell = headCells[i];
+    const headCellText = headCell.innerText;
+
+    if (headCellText in counts) {
+      headCell.innerHTML = `${headCellText} <span class='count'>(${counts[headCellText]})</span>`;
```

Comment:
Looks good and works as expected. As a nitpick, it might be nice to use `toLocaleString()` to make larger numbers more readable.
```suggestion
      headCell.innerHTML = `${headCellText} <span class='count'>(${counts[headCellText].toLocaleString()})</span>`;
```
Appearance in `en-US` locale:
<img width="481" alt="Screenshot 2024-06-12 at 3 51 59 PM" src="https://github.com/canonical/canonical-indico-themes/assets/46915153/c948e9c1-cc34-4593-afca-49e90c76f3a6">

Reply:
Good call, done :+1: